### ISSUE TITLE
[Forked due unattended]

### ISSUE THREAD

**Issue by busslina** [desktop-dart/duration#69]
https://github.com/busslina/dart-duration

Corrected some bugs

Include snippet:
```yaml
dependencies:
  duration:
      git:
        url: https://github.com/busslina/dart-duration.git
        ref: master
```

**Comment by tejainece** [comment]
Published `3.0.14`. https://github.com/desktop-dart/duration/commit/227f4e2facea2b5ecb156b83893f9d6f96349db0

**Comment by busslina** [comment]
I'm closing fork after merging PR https://github.com/desktop-dart/duration/pull/71
